llvm-spirv/test/image.ll

```diff
@@ -1,5 +1,3 @@
-; XFAIL: *
```

Comment:
Same here, the test passes in the upstream translator repo, if we are going to disable any of tests here, we need to submit a tracker to investigate the failures and re-enable those tests.